Assets/Scripts/DDAA/ReloadDDAA.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public sealed class ReloadDDAA
+{
+    // --------------------------------- //
+    // Singleton related implementation //
+    //----------------------------------//
+
+    private static ReloadDDAA instance = null;
+    private static readonly object padlock = new object();
+
+    private ReloadDDAA() { }
+
+    public static ReloadDDAA Instance
+    {
+        get
+        {
+            lock (padlock)
```

Comment:
I know this is a generic thing, but we should have comments on new stuff (to our team) like this.

Reply:
ah ok 👍